packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -360,6 +359,26 @@ export default class InlineContent extends Base {
         text: text.text || text
       }
     };
+    if (textOverrideStyles?.textDecoration === 'line-through') {
+      const textWidth = measureTextWidth({
+        ...this.style.textStyle,
+        ...textOverrideStyles,
+        text: text.text || text
+      });
+      const strikethroughLine = {
+        rect: true,
+        w: textWidth - 2,
+        h: 2,
+        color: textOverrideStyles?.textColor || 0xffffffff,
+        y: textComponent.h / 2
```

Comment:
we may want to add `mountY: 0.5` so that the strikethrough is fully centered in the text

Reply:
The mountY doesnt work on my end to center. Did you test this out?

Reply:
i did, it's definitely minor 😆

but here's what i'm seeing. `mountY: 1` might be best? 
![Screenshot 2024-09-13 at 11 23 58 AM](https://github.com/user-attachments/assets/2cbeb2ae-4f87-4d3f-8c2e-74badd224dae)

Reply:
Just to throw my nitpicking two cents in, I think this looks best:
```js
h: textComponent.h * 0.08,
y: textComponent.h / 2,
mountY: 1
```

Reply:
Looks perfectly centered now. Thanks guys!